iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -659,19 +660,40 @@ popo::InterfacePortData* PortManager::acquireInterfacePortData(capro::Interfaces
     }
 }
 
+void PortManager::publishServiceRegistry() const noexcept
+{
+    if (m_serviceRegistryPublisherPortData.has_value())
```

Comment:
I think we should print at least a warning or even a call to the error handler when this is not set.

Reply:
Can this even happen by design?... I mean it should be created with the `Portmanager` and then live until its destruction, making the whole check only a performance hit...